www/layouts/about/section.html

```diff
@@ -173,17 +173,17 @@ <h1 class="text-center-mobile">
           class="background-color-highlight-red d-flex justify-content-center align-items-center h-100 px-3 py-5 p-lg-7"
         >
           <h2 id="ocw-20-text" class="font-helvetica-light font-weight-light text-white">
-            We've made great progress in our first 20 years - unlocking access
-            to knowledge, helping launch the OER movement and opening the
-            complete MIT curriculum to the world. And we'll keep growing and
-            evolving.
+            In our first 20 years, we’ve unlocked access to knowledge, 
+            helped launch the global OER movement, and 
+            opened the complete MIT curriculum to the world. And we'll 
+            keep growing and evolving.
             <br />
             <br />
-            We're excited to introduce our new platform - which you are using
-            now - to bring smartphone users into OCW's global community…provide
-            more powerful and insightful ways to discover just the content you
-            need… and be a foundation for new features and learning experiences
-            for years to come.
+            Our new platform, which you are now using, brings smartphone 
+            users fully into OCW's global community… provides more powerful 
```

Comment:
sorry, I mistakenly added a space here after the `...`. please remove. 
```suggestion
            users fully into OCW's global community…provides more powerful 
```